clients/imodelhub/src/test/FileHandlers.test.ts

```diff
@@ -255,7 +255,7 @@ describe("AzureFileHandler", async () => {
   }).timeout("30s");
 
   // ###TODO khanaffan This tends to hang on linux.
-  it.skip("should throw when tempfile is deleted", async () => {
+  it("should throw when tempfile is deleted", async () => {
```

Comment:
TBH @pmconne we should kill this entire package... It's sort of cruft we still build but don't/shouldn't need to anymore.

Reply:
nope, it can be a separate PR but just don't go out of your way to try and get these tests to pass if they're still flaky.